### DISCUSSION TITLE
GitHub Copilot for Xcode SwiftUI ?

### DISCUSSION THREAD

**Discussion by jclaude3** [community/community#50373] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

How to install on GitHub to write Swiftui code?

**Comment by Mohamed-SayedAlAhl** [comment]
GitHub Copilot is integrated into Visual Studio Code, but it is not officially available as a standalone tool or an extension for Xcode or SwiftUI. However, you can still use GitHub Copilot to generate code snippets and then copy and paste them into your Xcode or SwiftUI project.

OR 
you can use the non official  approach by following the instruction of this repo: 
https://github.com/intitni/CopilotForXcode

Hope this helps!!

**Comment by LACHKAR99ZAKARIA** [comment]
If you want to use GitHub Copilot with SwiftUI, you can integrate it seamlessly into your Swift development workflow using Visual Studio Code. Check out [this link](https://www.lopau.com/how-to-use-github-copilot-with-swift-using-vscode/) for more details. Happy coding!

**Comment by devm33** [comment, marked as answer]
## 👀  [GitHub Copilot code completion in Xcode is now available in public preview](https://github.com/orgs/community/discussions/141898) 🚀
